docs/configuration/rename_function.md

```diff
@@ -0,0 +1,69 @@
+# How to rename element in architecture object
+
+Architecture objects have callback, node, path, executor, and topic objects.  
+CARET serves APIs to update the names of these objects.
+
+The name of the target element in the architecture object is updated via the Python API.  
+Those update is also reflected in the architecture file outputted by [`arch.export`](./load_and_save.md#save).
+
+All of the following code snippets can be executed after load environment variables with `source /path/to/ros2_caret_ws/install/setup.bash`.
+
+## Rename callback_name
+
+You can update callback name from `src` to `dst` with `rename_callback` function in `Architecture` class.
```

Comment:
```suggestion
You can update callback names from `src` to `dst` with `rename_callback` function in `Architecture` class.
```
`name` is countable nouns. Other `name` must be changed.

Reply:
Fixed it in [d3600d3](https://github.com/tier4/CARET_doc/pull/159/commits/d3600d32534991b9eecdf156b005e361df174f6a).